applications/logan_progress_report/cover_letter.tex

```diff
@@ -0,0 +1,39 @@
+\documentclass{letter}
+\usepackage{hyperref}
+\signature{Zach Latta \\ Executive Director}
+\address{
+  hackEDU \\
+  44 Tehama Street \\
+  San Francisco, CA, 94105
+}
+\begin{document}
+
+\begin{letter}{
+    Richard Logan \\
+    Director \\
+    The Reva \& David Logan Foundation \\
+    980 N. Michigan Ave. \#1122 \\
+    Chicago, IL, 60611
+  }
+  \opening{Dear Mr. Richard Logan,}
+
+  I'm pleased to submit our mid-year report to the board of The Reva \& David
+  Logan Foundation. The attached document reviews our progress and outlines our
+  plans for the coming six months, with an eye on 2016 and beyond.
+
+  In brief: we created an early prototype of the ``club in a box'', grew from 36
+  to 47 schools, and doubled our presence in foreign countries. Our report
+  provides details, reflecting on our initial proposal and goals.
+
+  This upcoming semester we are focusing our attention on 12 clubs---an
+  initiative we are calling ``Labs''---to refine a working version of our ``club
+  in a box''. Once we've completed that, we will begin nationwide distribution
+  at our clubs and turn our attention to expansion.
+
+  If the board is pleased with our progress, we would like to request a renewal
+  of The Reva \& David Logan Foundation's contribution of \$50,000 to hackEDU to
+  support our operations through February 2016.
+
```

Comment:
I'm not sure exactly where the footer is, but when it was rendered for me the "Zach Latta, Executive Director" section did not seem very well aligned.

![9pfuo--mcjfaxk48dn4daa_w9ek1uzuht6v5wbbfoxg](https://cloud.githubusercontent.com/assets/8407370/9830107/1450fa1e-5965-11e5-9a2a-a9221d620735.png)